Makefile

```diff
@@ -60,3 +60,14 @@ nri-device-injector:
 	docker build --pull -t ${REGISTRY}/${DEVICE_INJECTOR_IMAGE}:${TAG} -f nri_device_injector/Dockerfile .
 
 .PHONY: all format test vet presubmit build container push partition-gpu
+
+bin/device-injector-test:
+	$(GO) test -c ./nri_device_injector -o ./bin/device-injector-test 
+
+.PHONY: device-injector-test
+device-injector-test: bin/device-injector-test
```

Comment:
Can we run this as part of the GitHub Actions workflow as well?

Reply:
Done